tmail-backend/mailbox/team-mailboxes/src/test/scala/com/linagora/tmail/team/TMailCanSendFromTest.scala

```diff
@@ -29,7 +29,7 @@ class TMailCanSendFromTest extends CanSendFromContract {
       integrationResources.getMailboxManager.getMapperFactory,
       integrationResources.getMailboxManager.getEventBus)
 
-    teamMailboxRepository = new TeamMailboxRepositoryImpl(integrationResources.getMailboxManager, subscriptionManager, new TeamMailboxExtensionBehaviourNoop)
+    teamMailboxRepository = new TeamMailboxRepositoryImpl(integrationResources.getMailboxManager, subscriptionManager, java.util.Set.of(new TeamMailboxCallbackNoop))
```

Comment:
Use an import for java.util.Set (you can name it JavaSet to avoid classpath clahes)